.buildkite/pipeline.yml

```diff
@@ -44,6 +44,30 @@ steps:
       - github_commit_status:
           context: Unit Tests
 
+  - label: ":microscope: Standalone WordPressAuthenticator Unit Tests"
+    command: |
+      echo "--- :rubygems: Setting up Gems"
+      install_gems
+
+      echo "--- :cocoapods: Setting up Pods"
+      install_cocoapods
+
+      echo "--- :swift: Setting up Swift Packages"
+      install_swiftpm_dependencies
+
+      echo "--- 🧪 Testing"
+      bundle exec fastlane run scan \
+        scheme:WordPressAuthenticator \
+        prelaunch_simulator:true \
+        device:'iPhone 16'
+    plugins: [$CI_TOOLKIT]
+    parallelism: 50
```

Comment:
As I understand, this line should be removed to prevent it running 50 times?

Reply:
Yep, agreed. Good catch.
```suggestion
```

Reply:
😓 good catch @staskus @AliSoftware I was too rushed in my cherry picking!